owl/EASE-OBJ.owl

```diff
@@ -2652,6 +2681,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#StructuralDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>A design of an object which describes its stability, strength and rigidity of structures, and considers the way in which parts of a system or object are arranged. 
```

Comment:
first time the term "system" appears. Maybe best to remove it here?
Anyways it is unclear why "system **or** object" -- why is object not a system?